monkey/monkey_island/Pipfile

```diff
@@ -36,6 +36,8 @@ pyyaml = "*"
 semver = "==2.13.0"
 flask-login = "*"
 flask-wtf = "*"
```

Comment:
Where is this package used?

Reply:
Leftover from some debugging. I will remove it.